hack/build-rpm-release.sh

```diff
@@ -67,7 +68,11 @@ name = OpenShift Origin Release from Local Source
 " > "${repo_path}/origin-local-release.repo"
 
 	os::log::info "Repository file for \`yum\` or \`dnf\` placed at ${repo_path}/origin-local-release.repo"
-	os::log::info "Install it with: "$'\n\t'"$ mv '${repo_path}/origin-local-release.repo' '/etc/yum.repos.d"
+	if [[ -n "${INSTALL_REPO:-}" ]]; then
+		sudo mv "${repo_path}/origin-local-release.repo" '/etc/yum.repos.d'
```

Comment:
If you're interacting with the repo only using `make` and want a one-stop-shop to build and install this, it helps.